model_analyzer/record/types/perf_latency.py

```diff
@@ -20,11 +20,12 @@
 @total_ordering
 class PerfLatency(Record):
```

Comment:
If we're going to make this the super class, then there should be no metric with the tag `perf_latency`

Reply:
The problem is the tag `perf_latency` is also used in user's config files. If `perf_latency` is changed to `perf_latency_p99`, this might break some existing config files. I'm not sure if this is a good idea.
@aramesh7 @Tabrizian What do you think?

Reply:
Maybe `perf_latency_p99` can be renamed, then set a deprecated tag `perf_latency` that will automatically populate `perf_latency_p99` with its value if `perf_latency_p99` is not specified.

Reply:
Yes we should have both tags point to the same metric if possible for now, and somehow show that the original tag is deprecated. Maybe a warning.

Reply:
This is done. https://github.com/triton-inference-server/model_analyzer/pull/187#discussion_r688810371